ts/components/ui/Switch.tsx

```diff
@@ -12,14 +13,18 @@ const maybeDisabled = fromPredicate(
  * NativeBase Switch component styled with the app's brand primary color
  */
 export default class Switch extends React.Component<NativeBase.Switch> {
-  public render() {
+  public render(accessibleLabelText) {
     const thumbColor: string = maybeDisabled(this.props.disabled)
       .map(_ => variables.brandPrimaryLight)
       .getOrElse(variables.contentPrimaryBackground);
 
     return (
       <NBSwitch
-        // Stick
+accessible={true}
+// added accessibleLabel to manage the text read by screen readers
+// this label can be a default value or received from external call
```

Comment:
We don't usually write comments about well-known or already documented behavior (ie: how optional properties work).